services/api/src/dao/project.js

```diff
@@ -145,7 +145,8 @@ const addProject = sqlClient => async (cred, input) => {
         },
         ${input.branches ? ':branches' : '"true"'},
         ${input.pullrequests ? ':pullrequests' : '"true"'},
-        ${input.production_environment ? ':production_environment' : 'NULL'}
+        ${input.production_environment ? ':production_environment' : 'NULL'},
+        ${input.auto_idle ? '"auto"':'"disabled"'}
```

Comment:
so whatever we send as a string in `auto_idle` it will never be added to the database?